src/derivative_operators/derivative_operator_functions.jl

```diff
@@ -38,52 +38,49 @@ for MT in [2,3]
             ndimsM = ndims(M)
```

Comment:
set this to dispatch only when dowind is false

Reply:
@ChrisRackauckas dispatching on false now, should be good to merge once tests pass